config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -9,6 +9,10 @@ presubmits:
       testgrid-create-test-group: "true"
       testgrid-num-failures-to-alert: "10"
     always_run: true
+    decorate: true
+    extra_refs:
```

Comment:
presubmits don't need the extra_refs for the repo they run on and I think adding it will result in undefined behavior

Reply:
Great, thanks! Removing!

Reply:
Removed.

@alvaroaleman if this is right, I can migrate all the jobs from sig-node here to podsubmits (it seems pretty 'trivial', as they use node configs).

Just don't know if there's a way to test like it is in production without actually merging it

Reply:
to my knowledge its not possible to test job changes on prow.k8s.io without actually merging them

Reply:
(You could copy the job, set optional: true on the copy, create a PR on k/k and check, then remove the copy and update the original here)

Reply:
Yeah, seems that's the only way: https://github.com/kubernetes/test-infra/issues/18338#issuecomment-659530553

Reply:
Damn, right then :)

So I can just migrated the required ``pull-kubernetes-node-e2e`` to not have a big change.

I've followed your idea and created a similar job, with other name to test if it works :)

If this is ok we can follow this way and then I can start migrating some other jobs to podutils